### PR TITLE
WIP: Add debian variables in controller and images

### PR DESCRIPTION
**Pull request by ktsamis**
## What does this PR change?

Add debian variables in controller and images, still in progress have to see what else is needed, this is an initial draft.

### CODE REVIEW

File: modules/controller/variables.tf [hunk 1]
```diff
@@ -337,6 +337,20 @@ variable "ubuntu2004_sshminion_configuration" {
   }
 }
 
+variable "debian9_minion_configuration" {
```
Comment (rjmateus) [inline modules/controller/variables.tf:340]: do you need to have a variable named `debian9_sshminion_configuration` also?

File: modules/controller/variables.tf [hunk 2]
```diff
@@ -337,6 +337,20 @@ variable "ubuntu2004_sshminion_configuration" {
...
+  description = "use module.<DEBIAN9_MINION>.configuration, see main.tf.libvirt-testsuite.example"
+  default = {
+    hostnames = []
+  }
+}
+
+variable "debian10_sshminion_configuration" {
```
Comment (rjmateus) [inline modules/controller/variables.tf:347]: do you need to have a variable named `debian10_minion_configuration` also?

File: salt/controller/bashrc [hunk 3]
```diff
@@ -50,8 +50,12 @@ export VIRTHOST_XEN_PASSWORD="linux" {% else %}# no Xen host defined {% endif %}
 {% if grains.get('ubuntu1604_sshminion') | default(false, true) %}export UBUNTU1604_SSHMINION="{{ grains.get('ubuntu1604_sshminion') }}" {% else %}# no UBUNTU1604 ssh minion defined {% endif %}
 {% if grains.get('ubuntu1804_minion') | default(false, true) %}export UBUNTU1804_MINION="{{ grains.get('ubuntu1804_minion') }}" {% else %}# no UBUNTU1804 minion defined {% endif %}
 {% if grains.get('ubuntu1804_sshminion') | default(false, true) %}export UBUNTU1804_SSHMINION="{{ grains.get('ubuntu1804_sshminion') }}" {% else %}# no UBUNTU1804 ssh minion defined {% endif %}
-{% if grains.get('ubuntu2004_minion') | default(false, true) %}export UBUNTU1804_MINION="{{ grains.get('ubuntu2004_minion') }}" {% else %}# no UBUNTU2004 minion defined {% endif %}
-{% if grains.get('ubuntu2004_sshminion') | default(false, true) %}export UBUNTU1804_SSHMINION="{{ grains.get('ubuntu2004_sshminion') }}" {% else %}# no UBUNTU2004 ssh minion defined {% endif %}
+{% if grains.get('ubuntu2004_minion') | default(false, true) %}export UBUNTU2004_MINION="{{ grains.get('ubuntu2004_minion') }}" {% else %}# no UBUNTU2004 minion defined {% endif %}
```
Comment (rjmateus) [inline salt/controller/bashrc:53]: These two lines were correct in another PR. I think you have mixed it.

**Review by rjmateus — CHANGES_REQUESTED** [review]
Some small questions

File: modules/controller/variables.tf (hunk 1 above)
Comment (ktsamis) [reply modules/controller/variables.tf:340]: Yes just added

File: modules/controller/variables.tf (hunk 2 above)
Comment (ktsamis) [reply modules/controller/variables.tf:361]: Yes just added

File: salt/controller/bashrc (hunk 3 above)
Comment (ktsamis) [reply salt/controller/bashrc:53]: Yes I will remove these from here

**Review by rjmateus — APPROVED** [review]
Yo should rebase against master to solve conflicts, and then I think it can be merged.

**Comment by ktsamis** [comment]
Should I merge this? I think maybe some salt changes are needed still but I don't know if I have the time for more of this today. Is this something that you can continue/check if more is needed?

**Pull request merged**